### ISSUE TITLE
BehaviorSpace: bad code in a plot causes bug dialog when experiment is run

### ISSUE THREAD

**Issue by SethTisue** [NetLogo/NetLogo#425]
normally having code in a plot that doesn't compile doesn't prevent a model run from happening in the GUI. but in a parallel BehaviorSpace experiment, you get e.g.:

```
NetLogo is unable to supply you with more details about this error.
Please report the problem [...]

Nothing named NEWINJ>0 has been defined at position 58 in 
 at org.nlogo.plot.PlotManager.compileAllPlots(PlotManager.scala:76)
 at org.nlogo.headless.HeadlessModelOpener.finish(HeadlessModelOpener.scala:91)
 at org.nlogo.headless.HeadlessModelOpener.openFromMap(HeadlessModelOpener.scala:75)
 at org.nlogo.headless.HeadlessWorkspace.openString(HeadlessWorkspace.scala:521)
 at org.nlogo.app.App$$anon$6.newInstance(App.scala:133)
 at org.nlogo.lab.gui.Supervisor$$anonfun$start$1.apply$mcVI$sp(Supervisor.scala:90)
 at scala.collection.immutable.Range.foreach$mVc$sp(Range.scala:81)
 at org.nlogo.lab.gui.Supervisor.start(Supervisor.scala:89)
```

the error should be reported to the user normally, rather than via a "please report the problem" type dialog

reported by David C. Bell

**Comment by atfornes** [comment]
I have experienced this issue in version 5.3-64 and it the "please report problem.." dialog is misleading.

**Comment by mrerrormessage** [comment]
Related: #929

**Comment by mrerrormessage** [comment]
This was likely fixed by #929  - re-verify before starting work.

**Comment by mrerrormessage** [comment]
This was not fixed by #929. It remains an outstanding bug.
